app/models/theme_field.rb

```diff
@@ -138,6 +138,9 @@ def process_extra_js(content)
         js_compiler.append_module(content, filename)
       when "hbs"
         js_compiler.append_ember_template(filename.sub("discourse/templates/", ""), content)
+      when "hbr", "raw.hbs"
+        js_compiler.append_raw_template(filename.sub("discourse/templates/", ""), content)
+        js_compiler.append_raw_template(filename.sub("discourse/templates/", ""), content)
       when "raw.hbs"
```

Comment:
You might want to remove this now 😉 

(yeah, GH's suggestions aren't working well on multi-line).

Reply:
thanksssss!